src/main/webapp/app/shared/import-list/import-table.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="d-flex align-items-center justify-content-between">
+    <h5 class="m-0" [jhiTranslate]="columnBaseTranslationKey() + '.table.title'"></h5>
+    <div>
+        <div class="d-flex align-items-center">
+            @if (isLoading()) {
+                <fa-icon [icon]="faSpinner" animation="spin" />
+            }
+            <input
+                id="import-objects-search"
+                type="search"
+                class="form-control ms-2 rounded"
+                [placeholder]="'global.importTable.searchPlaceholder' | artemisTranslate"
+                aria-label="Search"
+                [(ngModel)]="searchTerm"
```

Comment:
Can we get rid of the error here?
![image](https://github.com/user-attachments/assets/6ad6a634-2c1a-45e2-8fc7-7a42ef9edf93)

Reply:
This is no error, IntellJ just doesn't know it yet.

Reply:
Still it's underlined, confusing people that quickly scan the code or try to fix it. Is there some tag or comment (`#ignore' or similar) to fix that?

Reply:
Not really I think we just have to wait for the next small Intellj release as they are always one or two behind new Angular features.

Reply:
For me this is fine, I guess that this should be supported in the upcoming months by IntelliJs language service